### ISSUE TITLE
Option to not echom "Connecting to db..."

### ISSUE THREAD

**Issue by willruggiano** [kristijanhusak/vim-dadbod-completion#56]
In neovim with cmdheight=0 this forces a hit-enter-prompt.

I'd like the option to disable (or redirect) messages rather than echoing them.

**Comment by kristijanhusak** [comment]
That is coming from https://github.com/kristijanhusak/vim-dadbod-ui.
You can opt-in to use vim-notify with this option https://github.com/kristijanhusak/vim-dadbod-ui/blob/master/doc/dadbod-ui.txt#L710.

**Comment by willruggiano** [comment]
Yeah, I meant this one: https://github.com/kristijanhusak/vim-dadbod-completion/blob/master/autoload/vim_dadbod_completion.vim#L399
